src/FastRouteFactory.php

```diff
@@ -5,24 +5,24 @@
 use FastRoute\DataGenerator\GroupCountBased;
 use FastRoute\RouteCollector;
 use FastRoute\RouteParser\Std;
-use Psr\Container\ContainerInterface;
 
 class FastRouteFactory
 {
-    public function __invoke(ContainerInterface $container)
+    public function __invoke()
     {
+        // TODO may it be used via di?
```

Comment:
I don't think there's a need for it. That is fairly concrete adapter.

Reply:
Ok

Reply:
Deleted